tests/epyccel/modules/openmp.py

```diff
@@ -0,0 +1,276 @@
+    #$ omp parallel
+    in_parallel = omp_in_parallel()
+    #$ omp end parallel
+    return in_parallel
+
+@types ('bool')
+def test_omp_set_get_dynamic(dynamic_theads):
+    from pyccel.stdlib.internal.openmp import omp_set_dynamic, omp_get_dynamic
+    omp_set_dynamic(dynamic_theads)
+    return omp_get_dynamic()
+
+@types ('bool')
+def test_omp_set_get_nested(nested):
+    from pyccel.stdlib.internal.openmp import omp_set_nested, omp_get_nested
+    omp_set_nested(nested)
+    return omp_get_nested()
+
+def test_omp_get_cancellation():
+    from pyccel.stdlib.internal.openmp import omp_get_cancellation
+    cancel_var = omp_get_cancellation()
+    return cancel_var
+
+def test_omp_get_thread_limit():
+    from pyccel.stdlib.internal.openmp import omp_get_thread_limit
+    #$ omp parallel
+    maximum_threads_available = omp_get_thread_limit()
+    #$ omp end parallel
+    return maximum_threads_available
+
+@types ('int')
+def test_omp_get_set_max_active_levels(max_active_levels):
+    from pyccel.stdlib.internal.openmp import omp_get_max_active_levels, omp_set_max_active_levels
+    omp_set_max_active_levels(max_active_levels)
+    max_active_levels_var = omp_get_max_active_levels()
+    return max_active_levels_var
+
+def test_omp_get_level():
+    from pyccel.stdlib.internal.openmp import omp_get_level
+    #$ omp parallel
+    #$ omp parallel
+    nested_parallel_regions = omp_get_level()
+    #$ omp end parallel
+    #$ omp end parallel
+    return nested_parallel_regions
+
+def test_omp_get_active_level():
+    from pyccel.stdlib.internal.openmp import omp_get_active_level
+    #$ omp parallel
+    #$ omp parallel
+    active_level_vars = omp_get_active_level()
+    #$ omp end parallel
+    #$ omp end parallel
+    return active_level_vars
+
+def test_omp_get_ancestor_thread_num():
+    from pyccel.stdlib.internal.openmp import omp_get_ancestor_thread_num, omp_get_active_level
+    #$ omp parallel
+    active_level = omp_get_active_level()
+    ancestor_thread = omp_get_ancestor_thread_num(active_level)
+    #$ omp end parallel
+    return ancestor_thread
+
+def test_omp_get_team_size():
+    from pyccel.stdlib.internal.openmp import omp_get_team_size, omp_get_active_level
+    #$ omp parallel
+    active_level = omp_get_active_level()
+    team_size = omp_get_team_size(active_level)
+    #$ omp end parallel
+    return team_size
+
+def test_omp_in_final():
+    from pyccel.stdlib.internal.openmp import omp_in_final
+    x = 20
+    z = 0
+    result = 0
+
+    #$ omp parallel
+    #$ omp single
+    #$ omp task final (i >= 10)
+    for i in range(x):
+        z = z + i
+        if omp_in_final() == 1:
+            result = 1
+    #$ omp end task
+    #$ omp end single
+    #$ omp end parallel
+    return result
+
+def test_omp_get_proc_bind():
+    from pyccel.stdlib.internal.openmp import omp_get_proc_bind
+
+    bind_var = omp_get_proc_bind()
+    return bind_var
+
+#The function give som errors
```

Comment:
Can we mark this test as xfail?

Reply:
we didnt make any test for this function, because the functions omp_get_partition_num_places and omp_get_place_proc_ids give a compilation error, i commented this test function because codacy complain about unused variables and imports.